mypy/errors.py

```diff
@@ -349,6 +349,17 @@ def add_error_info(self, info: ErrorInfo) -> None:
             self.only_once_messages.add(info.message)
         self._add_error_info(file, info)
 
+    def clear_errors_in_targets(self, path: str, targets: Set[str]) -> None:
```

Comment:
To try to improve my own understanding of this issue: which of the tests was breaking?

Reply:
`testImportStarAddMissingDependency1` and `testImportStarAddMissingDependencyInsidePackage1` would fail without this.